### PR TITLE
Fb user profile

### PR DESCRIPTION
**Pull request by FadiBaker92**

### CODE REVIEW

**Review by rezahusain — APPROVED** [review]
Looks like your on the right track! Code looks clean and concise.

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -12,21 +12,15 @@
   <% end %>
 
   <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
+    <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:15]: Nice use of hidden inputs!

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:21]: Since this is not being used, we can either remove it or add default values :)

**Review by JoseMDO — APPROVED** [review]
Looks good!

**Pull request merged**